cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -2625,8 +2623,7 @@ pub(crate) fn emit(
                 _ => unreachable!(),
             };
 
-            let inst = Inst::xmm_cmp_rm_r(cmp_op, dst.to_reg(), RegMem::reg(lhs));
-            inst.emit(sink, info, state);
+            Inst::x64_ucomis(*size, dst.to_reg(), RegMem::reg(lhs)).emit(sink, info, state);
```

Comment:
Like we were talking about in the Cranelift meeting, I think we want to reduce these methods hanging off of `Inst`. We can probably just fit into the existing `match size` above, but if that doesn't work, I also started using this pattern:

https://github.com/bytecodealliance/wasmtime/blob/89419ec2671de863cc2a0743d29a094c91355a45/cranelift/codegen/src/isa/x64/inst/emit.rs#L3099-L3105